airflow/config_templates/default_airflow.cfg

```diff
@@ -370,10 +370,13 @@ default_wrap = False
 # on webserver startup
 update_fab_perms = True
 
+# Minutes of non-activity before logged out from UI
+# 0 means never get forcibly logged out
```

Comment:
Can you add some documentation about this option?   If there is no information about this feature in the documentation, very few people will be able to use it.

Reply:
@mik-laj - > where do you think the documentation should be added ? Maybe you can provide some pointers? Most of the config options are documented in comments of the default_airlfow.cfg. Any other specific place you think this documentation should be added (and where all the previous options are documented?)

Reply:
Related discussion: https://github.com/apache/airflow/pull/6952#discussion_r362102402

I think, we can add short description in 'security.rst' file or a new file in `howto` directory.